src/pageObject.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return `Conduit logo`;
+  }
 };
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  }
 
+  clickOnLogo() {
+    return `Click on the Conduit logo`;
+  }
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, header, slug) {
+    super(url, header);
+    this.slug = slug;
```

Comment:
Constructor should modify original url by adding slug at the end. So use `this.url = url + slug;`